docs/conf.py

```diff
@@ -54,7 +54,8 @@
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
+# ones. When extensions are removed or added, please update the
+# documentation guide in the development guide.
```

Comment:
Reference the exact file and section that should be updated, so a user does not have to figure that out on their own.